db/data/20220519090946_backfill_deferral_dates_for_hesa_trainees.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class BackfillDeferralDatesForHesaTrainees < ActiveRecord::Migration[6.1]
+  def up
+    Trainee.joins(:hesa_student).where(state: "deferred", defer_date: nil).each do |trainee|
```

Comment:
There's a `deferred` scope we could use here.

Reply:
i will never remember this!